File: CS/WindowsFormsApplication7/XtraForm1.cs
using System;
using DevExpress.XtraEditors;

namespace WindowsFormsApplication7
{
	public partial class XtraForm1 : XtraForm
	{
		public XtraForm1()
		{
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			XtraForm child = new XtraForm();
			child.Text = "ChildForm 1";
			child.MdiParent = this;
			child.Show();

			XtraForm child2 = new XtraForm();
			child2.Text = "ChildForm 2";
			child2.MdiParent = this;
			child2.Show();

			child.BringToFront();
		}
	}
}

File: VB/WindowsFormsApplication7/XtraForm1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors

Namespace WindowsFormsApplication7
	Partial Public Class XtraForm1
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim child As New XtraForm()
			child.Text = "ChildForm 1"
			child.MdiParent = Me
			child.Show()

			Dim child2 As New XtraForm()
			child2.Text = "ChildForm 2"
			child2.MdiParent = Me
			child2.Show()

			child.BringToFront()
		End Sub
	End Class
End Namespace